core/src/test/scala/unit/kafka/server/AbstractFetcherThreadTest.scala

```diff
@@ -69,19 +69,19 @@ class AbstractFetcherThreadTest {
     fetcherThread.addPartitions(Map(partition -> 0L))
 
     // wait until lag metric is present
-    TestUtils.waitUntilTrue(() => allMetricsNames(FetcherMetrics.ConsumerLag),
+    TestUtils.waitUntilTrue(() => allMetricNames(FetcherMetrics.ConsumerLag),
       "Failed waiting for consumer lag metric")
 
     // remove the partition to simulate leader migration
     fetcherThread.removePartitions(Set(partition))
 
     // the lag metric should now be gone
-    assertFalse(allMetricsNames(FetcherMetrics.ConsumerLag))
+    assertFalse(allMetricNames(FetcherMetrics.ConsumerLag))
 
     fetcherThread.shutdown()
   }
 
-  private def allMetricsNames = Metrics.defaultRegistry().allMetrics().asScala.keySet.map(_.getName)
+  private def allMetricNames = Metrics.defaultRegistry().allMetrics().asScala.keySet.map(_.getName)
```

Comment:
These name changes are not strictly part of this fix, I'm wondering if we can open a MINOR pr for these while having this PR focus on streams only (to avoid confusion).